app/Admin/Controllers/Management/AccessController.php

```diff
@@ -49,6 +50,31 @@ public function index(Content $content)
             'member' => User::get(['id', 'firstname', 'lastname']),
         ]);
 
+        // Define the filter for parents.
+        $grid->filter(function($filter) {
+            // Get all poles (all assos with children).
+            $poles = Asso::whereIn('id', function($query) {
+                return $query->select('parent_id')->from('assos');
+            })->pluck('name', 'id');
+
+            $filter->where(function ($query) {
+                $query->whereHas('asso', function ($query) {
+                    $query->where('parent_id', '=', $this->input);
+                });
+            }, 'Parent')->select($poles);
+        });
+
+        $grid->get()->parent()->sortable()->display(function () {
```

Comment:
Same question as previous comment. We might want to give "Pole" as a parameter to the `parent()` function.

Reply:
I fixed it in my last commit, and changed the collumn name to Pôle.